conda/recipes/pylibcugraph/meta.yaml

```diff
@@ -39,41 +39,40 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
     - {{ compiler('cxx') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake {{ cmake_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cudatoolkit ={{ cuda_version }}
-    - cudf ={{ minor_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% endif %}
     - cython >=0.29,<0.30
-    - libcublas {{ libcublas_host_version }}
-    - libcublas-dev {{ libcublas_host_version }}
     - libcugraph ={{ version }}
-    - libcurand {{ libcurand_host_version }}
-    - libcurand-dev {{ libcurand_host_version }}
-    - libcusolver {{ libcusolver_host_version }}
-    - libcusolver-dev {{ libcusolver_host_version }}
-    - libcusparse {{ libcusparse_host_version }}
-    - libcusparse-dev {{ libcusparse_host_version }}
-    - libraft ={{ minor_version }}
-    - libraft-headers ={{ minor_version }}
-    - pylibraft ={{ minor_version}}
+    - pylibraft ={{ minor_version }}
     - python
+    # TODO: Remove rmm once https://github.com/rapidsai/raft/pull/1656 is merged (fixes pylibraft use of rmm)
```

Comment:
@jakirkham FYI, this is the line where this `rmm` dependency existed before. This TODO just makes it clear that it's a workaround with an upstream fix in progress. I'd be fine merging this PR with this dependency still in place since it affects both CUDA 11 and CUDA 12.